src/lib/util/util.js

```diff
@@ -371,4 +371,12 @@ Util.exec = promisify(exec);
  */
 Util.sleep = promisify(setTimeout);
 
+/**
+ * Object with certain title case variants for words
+ * @since 0.5.0
+ * @type {Object<string, string>}
+ * @static
+ */
+Util.titleCaseVariants = { textchannel: 'TextChannel' };
```

Comment:
VoiceChannel, CategoryChannel, and GuildMember lives matter.

Reply:
God bless their souls. I'll do them now